### PR TITLE
Currency select

### PR DESCRIPTION
**Pull request by marinanicolai**
The menu disappears while hoovering over it.

### CODE REVIEW

File: src/components/CurrencySelect/Currency.styles.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Menu } from "antd";
+import styled from "styled-components";
+import "antd/dist/antd.min.css";
+
+export const DropdownContent = styled.div``;
+
+export const Icon = styled.div``;
+
+export const StyledMenuItem = styled((props) => <Menu.Item {...props} />);
```
Comment (nicolad) [inline src/components/CurrencySelect/Currency.styles.js:9]: this is cool

**Review by nicolad — APPROVED** [review]

**Comment by nicolad** [comment]
this looks great, feel free to merge

**Comment by nicolad** [comment]
🚢

File: src/components/CurrencySelect/Currency.styles.js (hunk 1 above)
Comment (marinanicolai) [reply src/components/CurrencySelect/Currency.styles.js:9]: It's not working while hoovering over the menu

Comment (nicolad) [inline src/components/CurrencySelect/Currency.styles.js:9]:
```suggestion
export const StyledMenuItem = styled(Menu.Item);
```

**Pull request merged**